guides/RELEASE-PROCESS.md

```diff
@@ -130,3 +130,16 @@ Example changes:
 
 - Renamed or removed classes, mixins, functions, placeholders, or global variables.
 - Major visual changes to existing components
+
+# Sketch Library Release Process
+
+1. Download the [Sketch file from GitHub](https://github.com/CMSgov/design-system/blob/master/design-assets/CMS-Design-System-UI-kit.sketch)
+1. Update the CMSDS Sketch file locally with your changes
+1. Update the .xml file in the following ways
+   - Update the item `title` with the version number
+     - Example: `CMS Design System UI kit - v1.1`
+   - Update `pubDate` using [RFC822 format](https://hackage.haskell.org/package/time-http-0.5/docs/Data-Time-Format-RFC822.html)
+     - Example: `Mon, 13 Apr 2020 15:11:00`
+   - Update `enclosure sparkle:version` number
+     - Example: `sparkle:version="1.1"`
+1. Create a PR with the updates to the .xml file and Sketch file
```

Comment:
A little more detail here on creating a PR would be useful, assuming we'll have folks who've never done this before. Things like:
- How do you create a pull request?
- How do you include files in a PR? Do you upload them? Should you have started a new branch first?
- What content should they include in the PR?
- Who should they assign as reviewers? Is that automatic?
- What happens after the PR gets merged? (i.e. how updates should automatically go out to all who are using the .xml file)
- Are they responsible for communicating the new release? Where do they communicate that?

Reply:
> - How do you create a pull request?

linked to info on creating a PR

> - How do you include files in a PR? Do you upload them? Should you have started a new branch first?

linked to info on creating a PR in our contributing doc

> - What content should they include in the PR?

added 

> - Who should they assign as reviewers? Is that automatic?

No action needed the team will handle it

> - What happens after the PR gets merged? (i.e. how updates should automatically go out to all who are using the .xml file)

Still working this out 😅 

> - Are they responsible for communicating the new release? Where do they communicate that?

No, they are not responsible - the core team will communicate that in the release notes